docs/api/using-oauth.rst

```diff
@@ -0,0 +1,164 @@
+Using OAuth
+===========
+
+`OAuth <https://en.wikipedia.org/wiki/OAuth>`_ is an open standard that enables
+users to grant an application (an "OAuth client") the ability to interact with a
+service as that user, but without providing their authentication credentials
+(eg. username and password) directly to that application.
+
+OAuth clients follow an authorization process, at the end of which they get an
+access token which is included in API requests to the service. This process
+works as follows:
+
+1. The user clicks a login button or link in the application.
+
+2. The application sends them to an authorization page from the h service
+   (``/oauth/authorize``) via a redirect or popup, where the user will be
+   asked to approve access.
+
+3. If they approve, the authorization endpoint will send an authorization code
+   back to the application (via a redirect).
+
+4. The application exchanges the authorization code for a pair of tokens using
+   the h service's ``POST /api/token`` endpoint: A short-lived access token to
+   authorize API requests, and a long-lived refresh token.
+
+5. When the access token expires, the application obtains a new access token
```

Comment:
The application obtains the new access token using the refresh token, right? That’s not entirely clear to me from this documentation.

Reply:
Yes indeed. I'll clarify this.